src/content/introduction.md

```diff
@@ -40,8 +40,12 @@
 
 ---
 
+![Redux Flow](content/images/full-redux-flow.png "Redux Flow")
```

Comment:
I wish all these diagrams were the wizard + dragon diagram with the selectors step included ❤️